.github/dependabot.yaml

```diff
@@ -1,10 +1,11 @@
-version: 2
-updates:
-  - package-ecosystem: "yarn"
-    directories:
-      - "."
-    schedule:
-      interval: "weekly"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
\ No newline at end of file
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
Aurora Core uses yarn?

Reply:
`npm` is just how dependabot calls the ecosystem that uses a `package.json`, so this still work if the repo uses yarn.